core/src/androidMain/kotlin/BluetoothLeScannerAndroidScanner.kt

```diff
@@ -75,6 +76,8 @@ internal class BluetoothLeScannerAndroidScanner(
             }.build()
         }
 
+        checkBluetoothAdapterState(STATE_ON)
```

Comment:
Move right before the `startScan` call? i.e. just above line 88?

Reply:
Can do, wasn't sure which side of the log this should live on

Reply:
Ya, I often have trouble deciding too.
I think in this case though, it would make sense to have a log of what we're about to do right before a crash — I think it is helpful information in this case.